src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -1845,19 +1845,37 @@ public T Min
         {
             get
             {
-                T ret = default(T);
-                InOrderTreeWalk(delegate (SortedSet<T>.Node n) { ret = n.Item; return false; });
-                return ret;
+                if (_root == null)
+                {
+                    return default(T);
+                }
+
+                var current = _root;
```

Comment:
Nit: please replace ```var``` with ```Node```. We try to only use ```var``` when the type is obvious from the right-hand side of the assignment.